frontend/projects/upgrade/src/app/features/dashboard/home/components/enrollment-over-time/enrollment-over-time.component.ts

```diff
@@ -203,16 +217,24 @@ export class EnrollmentOverTimeComponent implements OnChanges, OnInit, OnDestroy
           });
         }
       });
+
+      // Sort the series array alphabetically based on the condition names.
+      // This ensures consistent ordering of conditions in the resulting graph.
+      series.sort((a, b) => a.name.localeCompare(b.name));
+
       return {
         name:
-          this.selectedDateFilter === DATE_RANGE.LAST_SEVEN_DAYS
-            ? this.dateToString(new Date(graphData.date), days)
-            : months[new Date(graphData.date).getMonth()],
+        this.selectedDateFilter === DATE_RANGE.LAST_SEVEN_DAYS
```

Comment:
Indentation is not right. (Prettier errors)

Reply:
Fixed indentation with Prettier